packages/lesswrong/lib/collections/digestPosts/schema.ts

```diff
@@ -17,6 +17,7 @@ const schema: SchemaType<DbDigestPost> = {
     canCreate: ['admins'],
     canUpdate: ['admins'],
     hidden: true,
+    nullable: false
```

Comment:
Pretty sure this entire collection is only used by the EA-forum, so the process which generated the list of fields to be updated wouldn't necessarily be reliable here.  EA forum should review changes to this file